ai-platform-unified/notebooks/official/automl/automl-text-classification.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<blockquote>Before you can view the model evaluation you must first list all of the evaluations for that model. Each model can have multiple evaluations, although new model is likely to only have one.</blockquote><p>I'm not sure if I understand this part. My understanding is that you can list aggregated evaluation metrics for the model, metrics for each label, or metrics for just one label. I believe, in this example, we're showing the aggregated metrics (for all labels) @ different confidence thresholds.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Yes, I think that's right. From a user's perspective, each model maps to n evaluations, where some are the aggregated values, some are for labels, etc. That said, I can't recall seeing a model that has more than one evaluation (when you list a model's evaluations).</p>

Reply:
<p>Right, that's where I'm a bit confused. Before even viewing multiple evaluations, how do users generate multiple evaluations? If that's possible, we should probably document this (in the user guide not the notebook).</p>